### PR TITLE
feat(sdk): Merge python and rust stacktraces

### PR DESCRIPTION
**Pull request by jan-auer**
Merges rust backtraces from libsymbolic or libsemaphore with the python traceback before sending events to Sentry.

screenshot 2018-11-28 at 00 45 23

 - [x] Parse stacktraces from `rust_info`
 - [x] Append them to python before sending
 - [x] Migrate to integration instead of `before_send` hook
 - [x] Make sure stuff still prints nicely on stdout

### CODE REVIEW

File: src/sentry/utils/rust.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        return event
+
+    exc_type, exc_value, tb = hint['exc_info']
+    if not hasattr(exc_value, 'rust_info') or not exc_value.rust_info:
+        return event
+
+    exception = get_path(event, 'exception', 'values', 0)
+    stacktrace = get_path(exception, 'stacktrace', 'frames')
+    if not stacktrace:
+        return
```
Comment (untitaker) [inline src/sentry/utils/rust.py]: Probably meant to `return event` here? This will drop the event.

**Review by untitaker — APPROVED** [review]
lgtm except that one `return` and the one below it

File: src/sentry/utils/sdk.py [hunk 2]
```diff
@@ -100,7 +101,8 @@ def capture_event(event):
         integrations=[
             DjangoIntegration(),
             CeleryIntegration(),
-            LoggingIntegration(event_level=None)
+            LoggingIntegration(event_level=None),
+            RustInfoIntegration(),
```
Comment (markstory) [inline src/sentry/utils/sdk.py]: Is this something that other customers could find useful as well?

Comment (mitsuhiko) [reply src/sentry/utils/sdk.py]: Yes but sadly it depends on our own exception types since there is no standard in python to carry foreign stacktraces ;(

Comment (markstory) [reply src/sentry/utils/sdk.py]: Boourns.

**Pull request merged**